.github/workflows/lambaa-release.yml

```diff
@@ -0,0 +1,32 @@
+name: Lambaa Release
+
+on:
+    push:
+        branches: master
+        paths:
+            - "src/**"
+            - "package.json"
+            - "package-lock.json"
+
+jobs:
+    create-release:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+              with:
+                  fetch-depth: 0
+            - uses: actions/setup-node@v2
+            - name: Generate tag name
+              id: tag_name
+              run: echo "::set-output name=tag_name::$(node -e 'console.log(require("./package.json").version)')"
+            - uses: johnyherangi/create-release-notes@v1.0.0
+              id: create_release_notes
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            - uses: actions/create-release@v1
+              env:
+                  GITHUB_TOKEN: ${{ secrets.RELEASE_TOKEN }}
+              with:
+                  tag_name: ${{ steps.tag_name.outputs.tag_name }}
+                  release_name: ${{ steps.create_release_notes.outputs.release-name }}
```

Comment:
This will be the latest PR title from the list of commits in the release e.g. `Do stuff (#123)`. Could also just make the release name the same as the tag name?

Reply:
It looks like the tag name would just be the version? I'm in favour of that.

Reply:
```suggestion
                  release_name: ${{ steps.tag_name.outputs.tag_name }}
```